src/picongpu/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -55,7 +55,7 @@ using DefaultParticleAttributes = MakeSeq_t<
     position< position_pic >,
     momentum,
     weighting
-#if( ENABLE_RADIATION == 1 )
+#if( PARAM_RADIATION == 1 )
```

Comment:
QUESTION:
Why is this in the default `speciesDefinition.param` and not only in the KelvinHelmholtz example?